crates/aide/src/axum/mod.rs

```diff
@@ -580,7 +584,7 @@ where
     /// to pass on the API documentation from the nested service as well.
     pub fn nest_service<T>(mut self, path: &str, svc: T) -> Self
     where
-        T: Service<Request<Body>, Error = Infallible> + Clone + Send + 'static,
+        T: Service<Request<Body>, Error = Infallible> + Clone + Send + 'static + std::marker::Sync,
```

Comment:
why add new Sync trait bound?

Reply:
Tbh. It's what the compiler complained about

Reply:
@JakkuSakura https://github.com/tokio-rs/axum/pull/2473 explains it